requirements.txt

```diff
@@ -0,0 +1,4 @@
+demands==1.0.6
```

Comment:
These probably should be minimum version requirements instead of exact requirements, and we should also specify the same minimum versions in `setup.py`